packages/shared/prisma/schema.prisma

```diff
@@ -115,6 +115,7 @@ model Project {
   updatedAt           DateTime               @default(now()) @updatedAt @map("updated_at")
   deletedAt           DateTime?              @map("deleted_at")
   name                String
+  retentionDays       Int?                   @map("retention_days")
```

Comment:
@maxdeichmann Any concerns about placing this column on the projects table? I considered a separate `data_retention` table, but IMO there are no advantages to it.
We don't have to keep any state for retentions so the number of days should be perfectly sufficient.